mava/wrappers/graph_wrapper.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2022 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Note this is only here until this is merged into jumanji
+# PR: https://github.com/instadeepai/jumanji/pull/223
+
+from functools import cached_property
+from typing import Tuple, Union
+
+import chex
+import jax
+import jax.numpy as jnp
+import jraph
+from jumanji import specs
+from jumanji.env import State
+from jumanji.types import TimeStep
+from jumanji.wrappers import Wrapper
+
+from mava.types import GraphsTuple, MarlEnv, Observation, ObservationGlobalState
+
+
+class GraphWrapper(Wrapper):
+    """Wrapper to convert the observation structure from the environment into a Jraph GraphsTuple
+    representing the graph topology of the agents. The default is a fully-connected graph.
```

Comment:
Can you elaborate here a bit please when you say fully-connected graph do you mean each agent shares info with all other agents? Adding something like the docstring for `add_graph_to_observations` would be great!